airdrop/job/reward_processors/nunet_reward_processor.py

```diff
@@ -81,12 +83,27 @@ def __batch_insert(self, values, is_audit_row, force=False):
         if(len(values) > 0):
             rows.append(tuple(values))    
 
+    def __send_slack_message(self, message, send=False):
+        self.__slack_messages.append(message)
+        if send:
+            u = Utils()
+            for message in self.__slack_messages:
+                u.report_slack(type=0, slack_message=message, slack_config=SLACK_HOOK)
+
     @exception_handler(SLACK_HOOK=SLACK_HOOK, logger=logger)
-    def process_rewards(self):
-        rewards_query = "select address, min(total) as balance, min(staked) as staked, count(*) as occurrences " +\
-                        "from user_balance_snapshot " +\
-                        f"where airdrop_window_id = {self._window_id} and total >= {AGIX_THRESHOLD_IN_COGS} " +\
-                        "group by address "
+    def process_rewards(self, only_registered):
+        if only_registered:
+            self.__send_slack_message(f"Computing final rewards for window {self._window_id}")
+            rewards_query = "select address, min(total) as balance, min(staked) as staked, count(*) as occurrences " +\
+                            "from user_balance_snapshot " +\
+                            f"where airdrop_window_id = {self._window_id} and total >= {AGIX_THRESHOLD_IN_COGS} " +\
```

Comment:
AGIX_THRESHOLD_IN_COGS is this specific to a particular window / airdrop ? or can it be ?

Reply:
Specific to an airdrop. This file has to be created for each airdrop and seeded in the DB. Its dynamically executed